components/LottieAnimation/LottieAnimation.tsx

```diff
@@ -6,7 +6,7 @@ interface Props {
   direction?: 1 | -1;
   loop?: boolean;
 }
-export function LottieAnimation({ animationData, play, direction = 1, loop = true }: Props) {
+export default function LottieAnimation({ animationData, play, direction = 1, loop = true }: Props) {
```

Comment:
I usually don't like default exports much, but for using the dynamic import it makes things a lot cleaner. Else you'd have to do them like `dynamic(() => import("./Something.tsx").then(mod => mod.Something))` which is quite a pain